canonicalization.md

```diff
@@ -0,0 +1,20 @@
+# Canonicalization
+
+OCI Images [are](descriptor.md) [content-addressable](image-layout.md).
+One benefit of content-addressable storage is easy deduplication.
```

Comment:
Ah, I [stand corrected][1].  I still think the current wording covers that though.  Would you still prefer [your initial suggestion][2]?

[1]: https://github.com/opencontainers/image-spec/pull/339#issuecomment-251412226
[2]: https://github.com/opencontainers/image-spec/pull/339#discussion_r81730202